### PR TITLE
Removes rigsuits.

### PR DESCRIPTION
**Pull request by kyrahabattoir**
Commando, DeathSquad and Ninja might need some equipment tweaks.

This is a bit of a monster patch, but I had to try to make it work.
Take it if you want, or don't. Turns out that rig suits are a very big part of bay, somehow.

### CODE REVIEW

**Comment by mattroks101** [comment]
I'll look this over when I get home.

File: code/game/antagonist/outsider/commando.dm [hunk 1]
```diff
@@ -28,7 +28,11 @@ var/datum/antagonist/deathsquad/mercenary/commandos
 	player.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/syndicate(player), slot_wear_mask)
 	player.equip_to_slot_or_del(new /obj/item/weapon/storage/box(player), slot_in_backpack)
 	player.equip_to_slot_or_del(new /obj/item/ammo_magazine/box/c45(player), slot_in_backpack)
-	player.equip_to_slot_or_del(new /obj/item/weapon/rig/merc(player), slot_back)
+
+	//fix me commando need a suit!
```
Comment (kyrahabattoir) [inline code/game/antagonist/outsider/commando.dm]: This is not ideal, but at least they get a suit.

File: code/game/antagonist/outsider/deathsquad.dm [hunk 2]
```diff
@@ -47,7 +47,11 @@ var/datum/antagonist/deathsquad/deathsquad
 		player.equip_to_slot_or_del(new /obj/item/weapon/plastique(player), slot_l_store)
 	player.equip_to_slot_or_del(new /obj/item/weapon/gun/projectile/revolver/mateba(player), slot_belt)
 	player.equip_to_slot_or_del(new /obj/item/weapon/gun/energy/pulse_rifle(player), slot_r_hand)
-	player.equip_to_slot_or_del(new /obj/item/weapon/rig/ert/assetprotection(player), slot_back)
+
+	//fix me! deathsquad need a suit!
```
Comment (kyrahabattoir) [inline code/game/antagonist/outsider/deathsquad.dm]: This is not ideal, but at least they get a suit.

File: code/game/antagonist/outsider/ninja.dm [hunk 3]
```diff
@@ -106,18 +106,10 @@ var/datum/antagonist/ninja/ninjas
...
-	player.equip_to_slot_or_del(ninjasuit,slot_back)
-	if(ninjasuit)
-		ninjasuit.toggle_seals(src,1)
-		ninjasuit.seal_delay = initial(ninjasuit.seal_delay)
-
-	if(istype(player.back,/obj/item/weapon/rig))
-		var/obj/item/weapon/rig/rig = player.back
-		if(rig.air_supply)
-			player.internal = rig.air_supply
+	player.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/syndicate(player), slot_wear_mask)
```
Comment (kyrahabattoir) [inline code/game/antagonist/outsider/ninja.dm]: This is kind of bad and by the look of it bay made the ninja a rigsuit killing machine as he doesn't get any weapons beside his special rig.

File: code/game/gamemodes/objective.dm [hunk 4]
```diff
@@ -521,21 +521,17 @@ datum/objective/download
 			return 0
 
 		var/current_amount
-		var/obj/item/weapon/rig/S
-		if(istype(owner.current,/mob/living/carbon/human))
-			var/mob/living/carbon/human/H = owner.current
-			S = H.back
-
-		if(!istype(S) || !S.installed_modules || !S.installed_modules.len)
-			return 0
```
Comment (kyrahabattoir) [inline code/game/gamemodes/objective.dm]: It appears tha they also don't accept tech diskettes anymore and you have to use the datajack, so this objective is broken.

File: code/modules/surgery/other.dm [hunk 5]
```diff
@@ -168,6 +168,7 @@
 
 	//no damage or anything, just wastes medicine
```
Comment (kyrahabattoir) [inline code/modules/surgery/other.dm]: I don't understand how surgery works with the different procedures, but this compiles.

**Pull request merged**